src/modelmanager.cpp

```diff
@@ -587,6 +588,7 @@ Status ModelManager::loadConfig(const std::string& jsonFilename) {
                 rapidjson::GetParseError_En(parseResult.Code()));
             intermediateStatus = StatusCode::JSON_INVALID;
             loud.log();
+            std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
What if administrator will save file with JSON schema error? I think with this loop we will have the config file reading retry and a sleep.

Reply:
It still could be that the file is only partially saved thus it is not correct from the rapidjson perspective.

I investigate the use of inotify mechanism which in theory should be the proper fix, but it will not make it for the 2021.4.1